### PR TITLE
Corporation & player cards revised

### PR DESCRIPTION
**Pull request by yzemaze**
Corporation cards
+ CSS-grid
+ herald in title (rounded square)
+ big symbol (#583)
+ better alignment of holdings
+ contrast of used tokens 15% => 50%
+ classes/ids where necessary/useful
+ table/tr/td alignments
+ thead & tbody (semantics! accessability/screen readers)
- render_president removed, because not used anymore (code was already incorporated/moved inline afaik)
- render_treasury dito

CSS
+ default styles for .card, .nowrap, .title, text-alignment and font-style/weight
+ default styles for tables, ths and tds within .card.

Player cards
+ synergies with corp card: a few props/styles removed, classes added
+ logo positioning in shares table changed
- table-wrapping divs removed

### CODE REVIEW

File: assets/app/view/game/corporation.rb [hunk 1]
```diff
@@ -14,20 +14,12 @@ def render
         end
 
         card_style = {
-          display: 'inline-block',
+          border: '1px solid gainsboro',
```
Comment (tobymao) [inline assets/app/view/game/corporation.rb:17]: should gainsboro and line 22 be current color as opposed to hardcoded to gainsboro and black?

File: assets/app/view/game/corporation.rb [hunk 2]
```diff
@@ -45,53 +37,46 @@ def render
...
-            'font-weight': 'bold',
-            'margin-left': 'auto',
-            'margin-right': 'auto',
-          },
-        }
-
         if @corporation.owner
           subchildren = (@corporation.operating_history.empty? ? [] : [render_revenue_history])
-          children << h(:table, revenue_table_props, [h(:tr, subchildren)])
+          children << h('table.revenue', [h(:tr, subchildren)])
```
Comment (tobymao) [inline assets/app/view/game/corporation.rb:42]: remove all of the CSS inheritance, make a method that does inline styles for the table components that you want

File: assets/app/view/game/corporation.rb [hunk 3]
```diff
@@ -45,53 +37,46 @@ def render
...
         end
 
-        h(:div, { style: card_style, on: { click: onclick } }, children)
+        h('div.corp.card', { style: card_style, on: { click: onclick } }, children)
```
Comment (tobymao) [inline assets/app/view/game/corporation.rb:45]: there's no reason to have a corp class

File: assets/app/view/game/corporation.rb (hunk 1 above)
Comment (yzemaze) [reply assets/app/view/game/corporation.rb:17]: For now it’s ok. I’ll revise this later on to fix issue #666 
currentColor only works for font color and would be wrong here. There’s no equivalent for bg – sadly.

File: assets/app/view/game/corporation.rb [hunk 4]
```diff
@@ -45,53 +37,46 @@ def render
...
-          background: @corporation.color,
-          color: @corporation.text_color,
-          'text-align': 'center',
-          padding: '0.5rem 0px',
-          'font-weight': 'bold',
-          margin: '-0.5rem -0.5rem 0 -0.5rem',
+        title_row_props = {
+          style: {
+            'background-color': @corporation.color,
+            color: @corporation.text_color,
```
Comment (tobymao) [inline assets/app/view/game/corporation.rb:52]: move back to inline style, i don't see much benefit to having a specific styling like token outside of here and in main.css

**Comment by yzemaze** [comment]
+ default style for a set, specific styles removed
+ some styles moved inline
+ main.css reorderd (a bit)

File: public/assets/main.css [hunk 5]
```diff
@@ -209,3 +196,38 @@ input#chatbar {
...
+  line-height: 1.25rem;
+}
+.card th {
+  padding: 0 0.3rem;
+}
+.card td {
+  padding: 0 0.5rem;
+}
+
+.corp div {
```
Comment (tobymao) [inline public/assets/main.css:225]: this seems leftover, should we put it back?

File: public/assets/main.css [hunk 6]
```diff
@@ -209,3 +196,38 @@ input#chatbar {
...
+  display: grid;
+  align-self: center;
+  justify-self: center;
+}
+table.shareholders, table.companies, table.revenue {
```
Comment (tobymao) [inline public/assets/main.css:230]: should we have these?

File: public/assets/main.css (hunk 5 above)
Comment (yzemaze) [reply public/assets/main.css:225]: .corp div { } is necessary to prevent using display: grid and alignment on every prop in corporation.rb

Comment (tobymao) [reply public/assets/main.css:225]: ok

File: public/assets/main.css (hunk 6 above)
Comment (yzemaze) [reply public/assets/main.css:230]: it saves three props

Comment (tobymao) [reply public/assets/main.css:230]: not worth it :D

Comment (yzemaze) [reply public/assets/main.css]: renamed class to center, moved collapse to prop

**Pull request merged**